walkthroughs/eks/getting-started-with-appmesh/cfn-output.json

```diff
@@ -0,0 +1,12 @@
+{
+  "PrivateSubnet1": "subnet-0af078e33ed7a7c90",
+  "PrivateSubnet2": "subnet-00ea93ea122d524c4",
+  "PrivateSubnets": "subnet-0af078e33ed7a7c90,subnet-00ea93ea122d524c4",
+  "VPC": "vpc-04f56a89912c8c359",
+  "PublicSubnets": "subnet-0df496d9368b4f341,subnet-0fc6f7a399ea88c05",
+  "PublicSubnet2": "subnet-0fc6f7a399ea88c05",
+  "StackRegion": "us-west-2",
+  "PublicSubnet1": "subnet-0df496d9368b4f341",
+  "StackName": "appmesh-service-discovery-eks",
+  "NodesSDPolicy": "arn:aws:iam::743088214495:policy/appmesh-service-discovery-eks-NodesSDPolicy-1DFM7X5R3ZJRA"
+}
```

Comment:
Guessing you didn't mean to commit this? Might be good to add to local .gitignore.

Reply:
Yeap, just adding a .gitignore here.